processor/src/main/java/com/instagram/common/json/annotation/processor/JsonParserClassData.java

```diff
@@ -404,14 +402,18 @@ private String generateExtractRvalue(TypeData data) {
       }
     }
 
-    return StrFormat.createStringFormatter(valueExtractFormatter)
-        .addParam("parser_object", "jp")
-        .addParam("subobject_class", data.getParsableType())
-        .addParam(
-            "subobject_helper_class",
-            data.getParsableTypeParserClass() +
-                JsonAnnotationProcessorConstants.HELPER_CLASS_SUFFIX)
-        .format();
+    StrFormat strFormat = StrFormat.createStringFormatter(valueExtractFormatter)
+            .addParam("parser_object", "jp")
+            .addParam("subobject_class", data.getParsableType());
+
+    if (!StringUtil.isNullOrEmpty(data.getParsableTypeParserClass())) {
```

Comment:
could you help to add some comments here about when/why getParsableTypeParserClass will be null?

Reply:
Yes. Would a comment on `getParsableTypeParserClass()` be preferable?

Reply:
(Y)